ha/util/conf_store.py

```diff
@@ -221,3 +221,20 @@ def get_disk_list_for_cvg(index, node_id, cvg_id):
         except Exception as e:
             Log.error(f"Unable to fetch Disk list from GConf. Error {e}")
             raise Exception(f"Unable to fetch Disk list. Error {e}")
+
+    @staticmethod
+    def get_control_pod(index):
+        """
```

Comment:
Please add output of SearchAPI as a comment here for better reference.

Reply:
Added the example i/p & o/p in the func docstring for easy understanding.
Please check now.

Reply:
Thanks!